src/extension/module.js

```diff
@@ -1112,7 +1130,15 @@ import sampleRUM from './rum.js';
     const { status } = sk;
     const resp = await sk.update();
     if (!resp.ok) {
-      if (!ranBefore) {
+      if (isRateLimited(resp)) {
+        sk.showModal({
+          message: i18n(sk, getServerError(resp).includes('onedrive')
```

Comment:
Could maybe write a convenience method for this since it looks like this condition is used in 7 other spots.

`getServerError(resp).includes('onedrive')`